tket/include/tket/Circuit/ConjugationBox.hpp

```diff
@@ -47,7 +47,7 @@ class ConjugationBox : public Box {
 
   Op_ptr symbol_substitution(
       const SymEngine::map_basic_basic &) const override {
-    return Op_ptr();
+    return std::make_shared<ConjugationBox>(compute_, action_, uncompute_);
   }
```

Comment:
`ConjugationBox` should support symbols, but we can fix it in a separate PR

Reply:
Made an issue and added a comment.